itou/templates/apply/list_for_job_seeker.html

```diff
@@ -1,67 +1,107 @@
+                Filtres des résultats {% include "includes/icon.html" with icon="filter" %}
+            </a>
+            <form method="get" action="" id="job-applications-filters-form" class="collapse show">
+                <div class="card c-card h-100 w-100 c-card--emploi">
+                    <div class="card-header">
+                        <h3>Filtrer les candidatures</h3>
+                    </div>
+                    <div class="card-body">
+                        {% include "apply/includes/job_applications_filter_statut.html" %}
+                        {% include "apply/includes/job_applications_filter_dates.html" %}
+                        {% include "apply/includes/job_applications_filter_pass.html" %}
+                    </div>
+                    <div class="card-footer text-right">
+                        <button id="js-job-applications-filters-apply-button" type="submit" class="btn btn-primary">Appliquer</button>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="col-12 col-md-8 mt-2">
+            <div class="clearfix">
+                <div class="float-left">
+                    <p class="h1">
+                        {% with job_applications_page.paginator.count as counter %}
+                            <span class="badge badge-emploi">{{ counter }}</span> <strong>Résultat{{ counter|pluralizefr }}</strong>
+                        {% endwith %}
                     </p>
-                    <p>{{ job_application.message|linebreaks }}</p>
-
-                    {% if job_application.state.is_refused and job_application.answer %}
-                        <p>
-                            <b>Réponse :</b>
-                        </p>
-                        <p>{{ job_application.answer|linebreaks }}</p>
-
-                    {% elif job_application.state.is_postponed and job_application.answer %}
-                        <p>
-                            <b>Motif de la mise en liste d'attente :</b>
-                        </p>
-                        <p>{{ job_application.answer|linebreaks }}</p>
+                </div>
+                <div class="float-right">
+                    {% if filters_counter > 0 %}
+                        <a href="{% url 'apply:list_for_job_seeker' %}" class="btn btn-outline-primary">
+                            Réinitialiser les filtres ({{ filters_counter }})
+                        </a>
                     {% endif %}
                 </div>
             </div>
-        </div>
-    {% endfor %}
 
-    {% include "includes/pagination.html" with page=job_applications_page %}
+            {% if not job_applications_page %}
+                <h2 class="h1 font-weight-normal text-muted">
+                    Aucune candidature pour le moment.
+                </h2>
+            {% endif %}
+
+            {% for job_application in job_applications_page %}
```

Comment:
Tiens, je suis étonnée qu'on ne fasse pas un `else` après le `if not job_applications_page`.